lego/apps/polls/tests/test_api_polls.py

```diff
@@ -113,17 +111,16 @@ def test_vote_poll_authenticated_answered(self):
         self.assertEqual(Poll.objects.get(pk=2).total_votes, votes)
         self.assertEqual(Poll.objects.get(pk=2).answered_users.count(), user_count)
 
-    def test_vote_poll_unauthenticated(self):
-        """A user with no permissions should be able to vote on a poll"""
-        self.client.force_authenticate(self.unauthenticated_user)
+    def test_vote_poll_unauthorized(self):
+        """A user with no permissions should not be able to vote on a poll"""
         votes = Poll.objects.get(pk=1).total_votes
         response = self.client.post(_get_vote_url(1), {"optionId": 1})
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(Poll.objects.get(pk=1).total_votes, votes + 1)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+        self.assertEqual(Poll.objects.get(pk=1).total_votes, votes)
 
     def test_vote_poll_not_valid(self):
         """A user should not be able to vote on a poll where valid_until is not in the future"""
-        self.client.force_authenticate(self.authenticated_user)
+        self.client.force_login(self.unauthenticated_user)
```

Comment:
This user should be renamed to `normal_user` or `pleb` (we mostly use pleb quite a bit). `unauthenticated_user` implies that there is no user (aka. not authenticated - Anonymous user https://docs.djangoproject.com/en/2.1/ref/contrib/auth/#anonymoususer-object).

Also, `self.authenticated_user` should be renamed to something like `admin`.

Maor info about authentication vs authorization: https://stackoverflow.com/questions/6556522/authentication-versus-authorization